src/OpenApi/src/Microsoft.AspNetCore.OpenApi.csproj

```
@@ -20,6 +20,7 @@
     <Reference Include="Microsoft.AspNetCore.Routing" />
     <Reference Include="Microsoft.AspNetCore.Mvc.Core" />
     <Reference Include="Microsoft.AspNetCore.Mvc.ApiExplorer" />
+    <Reference Include="Microsoft.AspNetCore.DefaultBuilder" />
```

```suggestion
    <Reference Include="Microsoft.AspNetCore" />
```

My bad. This should actually be `Microsoft.AspNetCore`. It's defined in the `src\DefaultBuilder` directory so I got my wires crossed.